ladder/teams/endpoint.yaml

```diff
@@ -1,6 +1,7 @@
 members:
 - aanm
 - christarazi
+- gandro
 - ianvernon
```

Comment:
We can probably drop @ianvernon from @cilium/endpoint too, they haven't been actively submitting changes in this area for a long time. The closer that these teams match the reality of the people investing time in maintaining the code, the better.

Reply:
The end of an era :'(